packages/contracts-bedrock/deploy-config/mainnet.json

```diff
@@ -1,36 +1,36 @@
 {
-  "finalSystemOwner": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
-  "controller": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
-  "portalGuardian": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
-  "l1StartingBlockTag": "0x3ed8369e0f4f28ecfdd5e2feebb945810a277def210bb36401a04cfbfaa9456d",
+  "finalSystemOwner": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
+  "controller": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
+  "portalGuardian": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
+  "proxyAdminOwner": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
+  "l1StartingBlockTag": "0x3ef7b122ab371d8d74e436740d17a8a428bc4fe5088117bf9602bcda57bf6dbe",
   "l1ChainID": 1,
   "l2ChainID": 10,
   "l2BlockTime": 2,
   "maxSequencerDrift": 600,
   "sequencerWindowSize": 3600,
   "channelTimeout": 300,
-  "p2pSequencerAddress": "0xAAAA45d9549EDA09E70937013520214382Ffc4A2",
+  "p2pSequencerAddress": "0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65",
   "batchInboxAddress": "0xff00000000000000000000000000000000000010",
-  "batchSenderAddress": "0x6887246668a3b87F54DeB3b94Ba47a6f63F32985",
+  "batchSenderAddress": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
   "l2OutputOracleSubmissionInterval": 1800,
-  "l2OutputOracleStartingTimestamp": 1685641931,
```

Comment:
we should delete the block tag, starting timestamp and starting number by setting them to `""` and `0`